src/components/Card.js

```diff
@@ -26,15 +54,30 @@ export default class Card {
   }
 
   _handleLikeIcon() {
+    if(this.isLiked()){
+      this._cardElement
+      .querySelector(".card__like-button")
+      .classList.add("card__like-button_active");
+    } else {
     this._cardElement
       .querySelector(".card__like-button")
-      .classList.toggle("card__like-button_active");
+      .classList.remove("card__like-button_active");
+    }
   }
-  _handleDeleteButton() {
+  
+  handleDeleteButton() {
```

Comment:
Function name should describe what the function does. This is not an event listener anymore, so you should use something like `deleteCard`

Reply:
Done